casper/src/main/scala/coop/rchain/casper/engine/BlockApproverProtocol.scala

```diff
@@ -201,7 +202,9 @@ object BlockApproverProtocol {
     (for {
       result                    <- EitherT(validate.pure[F])
       (blockDeploys, postState) = result
-      emptyStateHash            <- EitherT.right(RuntimeManager.emptyStateHashFixed.pure[F])
+      emptyStateHash <- EitherT.right(runtimeManager.spawnRuntime >>= { r =>
+                         r.preGenesisStateHash
+                       })
```

Comment:
Why not use hard-coded value here?

On the other hand, we can compare hard-coded value to computed and throw error if not equal.